examples/diffusion/laplace_iga_interactive.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+r"""
+Laplace equation with Dirichlet boundary conditions solved in a single patch
+NURBS domain using the isogeometric analysis (IGA) approach, using commands
+for interactive use.
+
+This script allows the creation of a customisable NURBS surface using igakit
+built-in CAD routines, which is then saved in custom HDF5-based files with
+.iga extension.
+
+View the results using::
+
+  $ ./postproc.py concentric_circles.vtk
+"""
```

Comment:
Please add:
```
import sys
sys.path.append('.')
```
- it is a hack that allows running the example from the source code directory on platforms, where the current directory (cwd) is not automatically added to sys.path (e.g. on a ecent kubuntu).